sources/packages/backend/apps/queue-consumers/src/processors/schedulers/workflow/assessment-workflow-enqueuer.scheduler.ts

```diff
@@ -41,14 +41,31 @@ export class AssessmentWorkflowEnqueuerScheduler extends BaseScheduler<void> {
       processSummary.info(
         "Checking application assessments to be queued for start.",
       );
-      // Process summary to be populated by the enqueueStartAssessmentWorkflows.
+      // Process summary to be populated by each enqueueing workflow call.
       // In case an unexpected error happen the finally block will still be able to
       // output the partial information captured by the processSummary.
-      const serviceProcessSummary = new ProcessSummary();
-      processSummary.children(serviceProcessSummary);
-      await this.workflowEnqueuerService.enqueueStartAssessmentWorkflows(
-        serviceProcessSummary,
-      );
+      try {
+        const serviceProcessSummary = new ProcessSummary();
```

Comment:
I would suggest having it renamed to `cancelProcessSummary` and the other to `startProcessSummary`.

The comment would then be updated to:`
```ts
// The cancelProcessSummary and startProcessSummary summaries are to be populated by each enqueueing child process call.
// In case an unexpected error happens the finally block will still be able to
 // output the partial information captured by the processSummary.
```

Reply:
I don't think it is applicable now as it was moved to `executeEnqueueProcess`.